docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -700,10 +700,42 @@ You can see them also by typing ```%%cython?`` in IPython or a Jupyter notebook.
 ============================================  =======================================================================================================================================
 
 
+.. _cython-cache:
+
+Cython cache
+============
+
+Cython cache is used to store cythonized ``.c``/``.cpp`` files to avoid running cython compiler on the files which were cythonized before.
```

Comment:
```suggestion
The Cython cache is used to store cythonized ``.c``/``.cpp`` files to avoid running the Cython compiler on the files which were cythonized before.
```